### DISCUSSION TITLE
Worth Interacting with Inverters?

### DISCUSSION THREAD

**Discussion by cstout1** [jasonacox/Powerwall-Dashboard#469] [Q&A]
I'm not sure if anybody has already done this or if it's worth pursuing, but I have two inverters and would like to see what each is generating throughout the day.

Why? So I could have a better idea and be quicker to act on a system failure like I just discovered today. My production graphs in powerwall dashboard went low during some cloudy weather, but after it cleared up and got sunny, the graphs stayed low for another week. Today, I discovered that one of my inverters hasn't been producing for two weeks and has a fault which will likely require replacement.

Anyone have thoughts on this?

**Comment by DerickJohnson** [comment]
I briefly looked into it when I first started using the dashboard. It might be worth looking into again, but at the time it was hard because my separate inverter wouldn’t even allow UI access unless you connected to it within a minute of rebooting it. I think the gateway might just aggregate the data together. Like you, I had issues with inverter comms and only knew thanks to the alerts. If you find anything interesting let me know! It might require a similar dual WiFi or network setup as people are trying to do with the gateways at the moment.
